### PR TITLE
Add recipe for blueconnect

### PR DESCRIPTION
**Pull request by DunderRoffe**
Add bitbake recipe for blueconnect which is a library for creating
bluetooth demos.

Signed-off-by: Viktor Sjölind <viktor.sjolind@pelagicore.com>

### CODE REVIEW

File: recipes-connectivity/blueconnect/blueconnect_git.bb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+DESCRIPTION = "Library for creating simple qt bluetooth demos"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
+
+DEPENDS = "qtbase qtdeclarative"
+RDEPENDS_${PN} = " \
+    qtbase \
+    qtdeclarative \
+    qtquickcontrols-qmlplugins \
+    ofono \
```
Comment (jonte) [inline recipes-connectivity/blueconnect/blueconnect_git.bb]: PKGCONFIG?

Comment (GordanM) [reply recipes-connectivity/blueconnect/blueconnect_git.bb]: Shouldn't you introduce the qmake configuration logic without ofono in the blueconnect repository and then add it in the recipe?

Comment (tobsan) [reply recipes-connectivity/blueconnect/blueconnect_git.bb]: I don't understand this comment. PKGCONFIG is for optionals. From how this is written, I suppose ofono is an absolute dependency for running the demo.

Comment (jonte) [reply recipes-connectivity/blueconnect/blueconnect_git.bb]: oFono is actually an optional dependency

**Comment by DunderRoffe** [comment]
@GordanM oh yeah forgot about that, will do :)

**Comment by DunderRoffe** [comment]
Added ofono as a RRECOMMENDS instead. It is really up to the application using the library if that functionality is used or not.

**Comment by DunderRoffe** [comment]
@GordanM I updated the recipe to follow the template but omitted the `PACKAGES` part since I wanna build all the stuffs. Is this OK? :)

**Pull request merged**